test/language/expressions/does-not-equals/bigint-and-bigint.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Non-strict inequality comparison of BigInt values
+esid: sec-abstract-equality-comparison
+info: |
+  1. If Type(x) is the same as Type(y), then
+    a. Return the result of performing Strict Equality Comparison x === y.
+
+  sec-numeric-types-bigint-equal
+  BigInt::equal (x, y)
+
+    The abstract operation BigInt::equal with two arguments x and y of BigInt type returns true if x and y have the same mathematical integer value and false otherwise.
+
+features: [BigInt]
+---*/
+
+assert.sameValue(0n != 0n, false);
```

Comment:
It would be really useful to add a message argument in the assertions so we can keep track on what is happening if any of these fails. The message can just reflect the actual argument being tested, e.g.:

```js
assert.sameValue(0n != 0n, false, "0n != 0n");
```

Otherwise a failing assertion during an implementation phase might be hard to map.